astropy/io/fits/tests/test_image.py

```diff
@@ -168,6 +168,53 @@ def test_open_3(self):
         finally:
             fits.conf.lazy_load_hdus = True
 
+    def test_fortran_array(self):
+        # Test that files are being correctly written+read for "C" and "F" order arrays
+        dims = (3,7)
+        size = np.product(dims)
+        a = np.zeros(dims, order='C')
+        b = np.zeros(dims, order='F')
+        a[:] = np.arange(size).reshape(*dims)
```

Comment:
This is test code, so certain amount of "magic numbers" is acceptable as long as they are not too magic and don't repeat. L173-176 are strictly unnecessary.
```suggestion
        a = np.arange(21).reshape(3, 7)
```